### PR TITLE
additions based on update

### PR DESCRIPTION
**Pull request by AnirudhG07**
Based on the recent PR [#1550](https://github.com/sxyazi/yazi/pull/1550), glow.yazi should also be updated to avoid any future complications.

Thanks

### CODE REVIEW

**Comment by AnirudhG07** [comment]
And also ya pack download instructions.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Plugin for [Yazi](https://github.com/sxyazi/yazi) to preview markdown files with [glow](https://github.com/charmbracelet/glow). To install, clone the repo inside your `~/.config/yazi/plugins/`:
```
Comment (Reledia) [inline README.md:3]: Remove completely the part related to cloning the repo and leave only the `ya pack` command

**Review by Reledia — CHANGES_REQUESTED** [review]
Little changes in the readme. Remove any mentions of cloning the repo, thanks

**Comment by AnirudhG07** [comment]
I hope this is fine. If any edits needed, you can change it suitable to u. 
Thanks

**Comment by Reledia** [comment]
It's perfect now

**Pull request merged**